packages/@webex/plugin-meetings/test/unit/spec/breakouts/breakout.ts

```diff
@@ -45,6 +51,11 @@ describe('plugin-meetings', () => {
 
         assert.equal(result, 'REQUEST_RETURN_VALUE');
       });
+      it('send metrics as expected', async () => {
+        Metrics.postEvent = sinon.stub();
+        await breakout.join();
+        assert.calledTwice(Metrics.postEvent);
```

Comment:
issue: It is important to check the arguments of the calls to Metrics.postEvent.